opm/autodiff/MultisegmentWell_impl.hpp

```diff
@@ -798,7 +902,7 @@ namespace Opm
                 const int sign = dwells[seg][SPres] > 0.? 1 : -1;
                 const double dx_limited = sign * std::min(std::abs(dwells[seg][SPres]), relaxation_factor * max_pressure_change);
                 // const double dx_limited = sign * std::min(std::abs(dwells[seg][SPres]) * relaxation_factor, max_pressure_change);
-                primary_variables_[seg][SPres] = old_primary_variables[seg][SPres] - dx_limited;
+                primary_variables_[seg][SPres] = std::max( old_primary_variables[seg][SPres] - dx_limited, 1e5);
```

Comment:
For this one, when negative BHP happens, it should switch to BHP control. Not sure we would like that. Similar code exists due to historical reason.